lib/src/widget/nav_bar/widget/nav_bar_container.dart

```diff
@@ -96,9 +98,15 @@ class NavBarContainerState extends State<NavBarContainer>
 
   @override
   void initState() {
-    MergeStream([resetStream.stream, updateStream.stream])
-        .debounceTime(updateNavBarDebounceTimeout)
-        .listen((bool? goingBack) {
+    final isInTest = Platform.environment.containsKey('FLUTTER_TEST');
+    Stream<bool?> stream =
+        MergeStream([resetStream.stream, updateStream.stream]);
+
+    if (isInTest) {
+      stream = stream.debounceTime(updateNavBarDebounceTimeout);
+    }
```

Comment:
🛑  Hmm this seems to be the opposite of what you wanted to achive:

missing `!`

Reply:
yeah... fixed